internal/pkg/manifest/validate.go

```diff
@@ -49,14 +62,21 @@ func (l *LoadBalancedWebServiceConfig) Validate() error {
 	}
 	for ind, taskDefOverride := range l.TaskDefOverrides {
 		if err = taskDefOverride.Validate(); err != nil {
-			return fmt.Errorf(`validate taskdef_overrides[%d]: %w`, ind, err)
+			return fmt.Errorf(`validate "taskdef_overrides[%d]": %w`, ind, err)
 		}
 	}
+	if err = validateDependencies(validateDependenciesOpts{
```

Comment:
nit: maybe instead of defining an additional type in the pkg
```suggestion
	if err = validateContainerDeps(containerName,  &l.ImageConfig.Image, l.Sidecars)
```